README.rst

```diff
@@ -1,62 +1,55 @@
+DataBiosphere Data Store CLI Client
+===================================
+
 This repository is a pip installable Command Line Interface (CLI) and Python library (API) for interacting with the
-Data Coordination Platform (DCP) of the Human Cell Atlas (HCA).
+DataBiosphere Data Store (DSS).
 
-Currently the `hca` package supports interaction with the `Upload Service <https://github.com/HumanCellAtlas/upload-service>`_ and `Data Storage Service (DSS) <https://github.com/HumanCellAtlas/data-store>`_ for services such as uploading, downloading,
-and querying data.
+Currently the `dbio` package supports interaction with the `Data Store <https://github.com/DataBiosphere/data-store>`_
+for uploading, downloading, and fetching information about data in the data store.
 
-The HCA CLI is compatible with Python versions 3.5+ (we are no longer compatible with Python 2.7, and our last compatible Python 2.7 version was `hca==6.4.0`).
+The DataBiosphere CLI is compatible with Python versions 3.5+.
 
 Installation
 ------------
-:code:`pip install hca`.
+
+:code:`pip install dbio`.
 
 Usage
 -----
 
 Documentation on readthedocs.io:
 
-* `CLI documentation <https://hca.readthedocs.io/en/latest/cli.html>`_
-
-* `Python API documentation <https://hca.readthedocs.io/en/latest/api.html>`_
-
-Example CLI/API usage:
+* `CLI documentation <https://dbio.readthedocs.io/en/latest/cli.html>`_
 
-* `CLI examples (open endpoints) <https://github.com/HumanCellAtlas/dcp-cli/tree/master/docs/OpenCLIExamples.rst>`_
+* `Python API documentation <https://dbio.readthedocs.io/en/latest/api.html>`_
 
-* `CLI examples (restricted endpoints) <https://github.com/HumanCellAtlas/dcp-cli/tree/master/docs/RestrictedCLIExamples.rst>`_
-
-* `Python API examples (open endpoints) <https://github.com/HumanCellAtlas/dcp-cli/tree/master/docs/OpenAPIExamples.rst>`_
-
-* `Python API examples (restricted endpoints) <https://github.com/HumanCellAtlas/dcp-cli/tree/master/docs/OpenAPIExamples.rst>`_
-
-To see the list of commands you can use, type :code:`hca --help`.
+To see the list of commands you can use, type :code:`dbio --help`.
 
 Configuration management
 ~~~~~~~~~~~~~~~~~~~~~~~~
-The HCA CLI supports ingesting configuration from a configurable array of sources. Each source is a JSON file.
+The DataBiosphere CLI supports ingesting configuration from a configurable array of sources. Each source is a JSON file.
 Configuration sources that follow the first source update the configuration using recursive dictionary merging. Sources
 are enumerated in the following order (i.e., in order of increasing priority):
 
-- Site-wide configuration source, ``/etc/hca/config.json``
-- User configuration source, ``~/.config/hca/config.json``
-- Any sources listed in the colon-delimited variable ``HCA_CONFIG_FILE``
+- Site-wide configuration source, ``/etc/dbio/config.json``
+- User configuration source, ``~/.config/dbio/config.json``
+- Data store specific configuration file, ``~/.config/dss/config.json``
```

Comment:
can omit this line, the dss config inside the user-config.

Reply:
Addressed by 2c25750